python/whylogs/extras/example.py

```diff
@@ -0,0 +1,165 @@
+import numpy as np
+from nltk.corpus import inaugural, stopwords
+from nltk.stem import PorterStemmer
+
+from whylogs.core.configs import SummaryConfig
+from whylogs.extras.nlp_metric import (
+    NlpConfig,
+    NlpMetric,
+    SvdMetric,
+    SvdMetricConfig,
+    UpdatableSvdMetric,
+    _preprocessifier,
+)
+
+# from nltk.tokenize import word_tokenize
+
+# import nltk
+# nltk.download('stopwords')
+# nltk.download('inaugural')
+
+# inverted index weighting utility functions
+
+
+def global_freq(A: np.ndarray) -> np.ndarray:
+    gf = np.zeros(A.shape[0])
+    for i in range(A.shape[0]):
+        for j in range(A.shape[1]):
+            gf[i] += A[i, j]
+    return gf
+
+
+def entropy(A: np.ndarray) -> np.ndarray:
+    gf = global_freq(A)
+    g = np.ones(A.shape[0])
+    logN = np.log(A.shape[1])
+    assert logN > 0.0
+    for i in range(A.shape[0]):
+        assert gf[i] > 0.0
+        for j in range(A.shape[1]):
+            p_ij = A[i, j] / gf[i]
+            g[i] += p_ij * np.log(p_ij) / logN if p_ij > 0.0 else 0.0
+    return g
+
+
+def log_entropy(A: np.ndarray) -> None:
+    g = entropy(A)
+    for i in range(A.shape[0]):
+        for j in range(A.shape[1]):
+            A[i, j] = g[i] * np.log(A[i, j] + 1.0)
+
+
+# the NLTK tokenizer produces punctuation as terms, so stop them
+stop_words = set(
```

Comment:
Does this put in any emoji or further special symbols? If so do those need stopped?